src/applications/edu-benefits/0994/pages/trainingProgramsInformation.js

```diff
@@ -82,8 +94,17 @@ export const schema = {
           providerName,
           programName,
           courseType,
-          location: {
-            ...location,
+          'view:location': {
+            type: 'object',
+            properties: {},
+            'ui:collapsed': true,
```

Comment:
Do you need this? It should be set to true when `checkLocation` returns false.

Reply:
without it the properties were showing before a `courseType` selection had been made